src/Auditable.php

```diff
@@ -143,6 +143,14 @@ protected function resolveAuditExclusions()
      */
     public function getAuditExclude(): array
     {
+        if(property_exists($this, 'auditExcludeMerge'))
+        {
+            return array_merge(
+                $this->auditExcludeMerge,
+                Config::get('audit.exclude', [])
+            );
+        }
```

Comment:
what about `bool $auditExcludeMerge `?
```php
if ($this->auditExcludeMerge ?? false) {
    return array_merge($this->auditExclude ?? [], Config::get('audit.exclude', []));
}
```
It seems excessive to me to have two properties for almost the same thing.
Also take a look at [PHP Coding Standards](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer)

Reply:
I like it!